Formula/mold.rb

```diff
@@ -49,7 +49,7 @@ def install
     # This helps make the bottle relocatable.
     inreplace "config.h.in", "@CMAKE_INSTALL_FULL_LIBDIR@", ""
     # Ensure we're using Homebrew-provided versions of these dependencies.
-    %w[mimalloc tbb zlib zstd].map { |dir| (buildpath/"third-party"/dir).rmtree }
+    %w[mimalloc tbb zlib zstd].each { |dir| (buildpath/"third-party"/dir).rmtree }
```

Comment:
This change doesn't really do anything, no?

Reply:
Right. Just some nitpicking. Since `rmtree` returns `nil`, it might be more semantically meaningful to use `each` here.

```plaintext
irb(main):001:0> require "pathname"
=> true
irb(main):002:0> files = Pathname.glob('*')
=> [#<Pathname:hello>, #<Pathname:world>]
irb(main):003:0> files.map(&:rmtree)
=> [nil, nil]
```

Reply:
We don't use the return value, so we should just leave this as-is. No need to add noise to `git blame` without a good reason.